### DISCUSSION TITLE
use `spring-cloud-aws-dependencies` directly for dependency management

### DISCUSSION THREAD

**Discussion by elkhart** [awspring/spring-cloud-aws#865] [Q&A]
I'm faced with introducing `Spring Cloud AWS` into our project and followed the reference docs

https://docs.awspring.io/spring-cloud-aws/docs/2.3.0/reference/html/index.html#basic-setup

But then I searched in our other company projects and found a lot of our projects would use the way recommended at

https://spring.io/blog/2021/03/17/spring-cloud-aws-2-3-is-now-available

which is using `spring-cloud-aws-dependencies` like

```
<dependencyManagement>
	<dependencies>
		<dependency>
			<groupId>io.awspring.cloud</groupId>
			<artifactId>spring-cloud-aws-dependencies</artifactId>
			<version>2.3.0</version>
			<type>pom</type>
			<scope>import</scope>
		</dependency>
	</dependencies>
</dependencyManagement>
```

I tried to use this way but then the [AWS SDK dependency version management](https://docs.awspring.io/spring-cloud-aws/docs/2.4.4/reference/html/index.html#amazon-sdk-dependency-version-management) doesn't seem to work.

In general I wonder if this is also a recommended way of configuring a project since it was mentioned in the release notes?

In case of a `yes` it would be nice to have this documented as well and in case of `no` it would be good to know what the downsides are in order to decide if all of our projects using this approach need to go the documented way.

**Comment by maciejwalkowiak** [comment]
@elkhart apologies for late answer. I recommend using latest version of Spring Cloud AWS that has completely new reference guide with getting started steps. In short - yes, you should use `spring-cloud-aws-dependencies` to set versions. 

https://docs.awspring.io/spring-cloud-aws/docs/3.0.3/reference/html/index.html#getting-started
